addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -594,5 +595,28 @@
     </div>
 </template>
 
+<template id="forum_related_posts" name="Related Posts">
```

Comment:
Maybe just "related_posts" ? As the template is prefixed by "website_forum" already?